examples/basic-api/src/Main.php

```diff
@@ -10,8 +12,13 @@ class Main {
     public function setCurrentToken($token) {
 
         try {
-            $this->tokenInfo = \Auth0\SDK\Auth0JWT::decode($token, getenv('AUTH0_CLIENT_ID'), getenv('AUTH0_CLIENT_SECRET'));
-            $this->token = $token;
+          $verifier = new JWTVerifier([
+              'valid_audiences' => [getenv('AUTH0_CLIENT_ID')],
+              'client_secret' => getenv('AUTH0_CLIENT_SECRET')
```

Comment:
Not really. New clients does not have base 64 encoded secrets so it is ok. But we should add a note and show how to use encoded secrets